script.js

```diff
@@ -39,13 +39,26 @@ class App extends React.Component {
 
   }
   signIn(email, password) {
-    // sign in APIs here
-    this.setState({
-      user: {
-        email,
-        password } });
-
-
+		axios.post('http://localhost:3000',{
```

Comment:
this does not appear to target the correct endpoint. When I try it I get a 404.

Reply:
Endpoint corrected, and after I said I would wait for the server's endpoint to change before pulling this, very clever of me